feed.rss

```diff
@@ -6,8 +6,8 @@
     <description>A podcast about Apple, Swift and other programming topics. By Benedikt Terhechte and Bas Broek.</description>
     <docs>http://blogs.law.harvard.edu/tech/rss</docs>
     <language>en</language>
-    <pubDate>Thu, 22 Oct 2020 16:14:25 GMT+2</pubDate>
-    <lastBuildDate>Thu, 22 Oct 2020 16:14:25 GMT+2</lastBuildDate>
+    <pubDate>Thu, 29 Oct 2020 12:48:45 GMT+2</pubDate>
+    <lastBuildDate>Thu, 29 Oct 2020 12:48:45 GMT+2</lastBuildDate>
```

Comment:
Should check this as we now have a different timezone. This has never really been made for it. Technically, it should not affect episodes that were in summer time, but 🤷‍♂️ 😛